src/methods/modular_programs/medium/q5/DivSeries.java

```diff
@@ -0,0 +1,28 @@
+package methods.modular_programs.medium.q5;
+
+/**
+ * *Write a Java program with a method called divisionCalc to ask the user for a
```

Comment:
Hey, I think the way this question text is laid out is confusing to the student. Could you please change it so is clearer what parts belong in the method and what parts go in main? Thanks.

Reply:
Also, please specify the parameters and return type of the method.